examples/docs/src/components/PageContent/PageContent.astro

```diff
@@ -2,9 +2,8 @@
 import MoreMenu from "../RightSidebar/MoreMenu.astro";
 import TableOfContents from "../RightSidebar/TableOfContents.tsx";
 
-const { content, githubEditUrl } = Astro.props;
+const { content, headings, githubEditUrl } = Astro.props;
 const title = content.title;
-const headings = content.astro.headings;
```

Comment:
`content.astro.headings` now throws! You'll get a nice error recommending `Astro.props.headings` instead 👍